### PR TITLE
Stub sys_io for vsh.

### PR DESCRIPTION
**Pull request by clienthax**

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_io.cpp [hunk 1]
```diff
@@ -11,26 +11,55 @@ error_code sys_io_buffer_create(u32 block_count, u32 block_size, u32 blocks, u32
 {
 	sys_io.todo("sys_io_buffer_create(block_count=0x%x, block_size=0x%x, blocks=0x%x, unk1=0x%x, handle=*0x%x)", block_count, block_size, blocks, unk1, handle);
 
-	return CELL_OK;
+	if (auto io = idm::make<lv2_io_buf>(block_count, block_size, blocks, unk1))
+	{
+		*handle = io;
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_io.cpp:16]: handle might be null though

File: rpcs3/Emu/Cell/lv2/sys_io.cpp [hunk 2]
```diff
@@ -11,26 +11,55 @@ error_code sys_io_buffer_create(u32 block_count, u32 block_size, u32 blocks, u32
...
 {
 	sys_io.todo("sys_io_buffer_allocate(handle=0x%x, block=*0x%x)", handle, block);
 
-	return CELL_OK;
+	if (auto io = idm::get<lv2_io_buf>(handle))
+	{
+		// no idea what we actually need to allocate
+		if (u32 addr = vm::alloc(io->block_count * io->block_size, vm::main))
+		{
+			*block = addr;
```
[the remark this answers is not in the document]
Comment (clienthax) [reply rpcs3/Emu/Cell/lv2/sys_io.cpp:46]: LV2 has no checks for block

[the remark this answers is not in the document]
Comment (clienthax) [reply rpcs3/Emu/Cell/lv2/sys_io.cpp:46]: aha its checked in the parent method.. 1m

[the remark this answers is not in the document]
Comment (Megamouse) [reply rpcs3/Emu/Cell/lv2/sys_io.cpp:51]: Needs a comment here at least

[the remark this answers is not in the document]
Comment (Megamouse) [reply rpcs3/Emu/Cell/lv2/sys_io.cpp:51]: Ah ok

**Review by elad335 — APPROVED** [review]

**Pull request merged**